content/200-concepts/100-components/03-prisma-migrate/010-mental-model.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: 'A mental model for Prisma Migrate'
+metaTitle: 'A mental model for Prisma Migrate'
+metaDescription: 'A mental model guide for working with Prisma Migrate in your project'
+tocDepth: 3
+---
+
+<TopBlock>
+
+This guide provides a conceptual overview of database migrations using Prisma Migrate when working with relational databases. It covers: what database migrations are, their value, and what Prisma Migrate is and how you can evolve your database schema with Prisma Migrate in different environments.
+
+**This guide does not apply to MongoDB** because Prisma Migrate only supports relational databases.
```

Comment:
`db push` is part of Migrate, but not part of the `migrate` commands. This might be to broad.
(Unless the guide really only covers `migrate` commands - but then this is not really a mental model for Prisma Migrate, but part of it)

Reply:
FWIW, I don't think of `prisma db push` belonging to Prisma Migrate. To me, it's just part of the Prisma CLI but Prisma Migrate IMO only encompasses `prisma migrate` commands indeed. It's a product architecture question I guess, but to me thinking of Prisma Migrate only in terms of the actual `prisma migrate` commands simplifies the product architecture.